cabal-install/src/Distribution/Client/Setup.hs

```diff
@@ -637,6 +639,14 @@ configureExOptions _showOrParseArgs src =
       (reqArg "VERSION" (parsecToReadE ("Cannot parse cabal lib version: "++)
                                     (fmap toFlag parsec))
                         (map prettyShow. flagToList))
+  , option "" ["append"]
+      "appending the new config to the old config file"
+      configAppend (\v flags -> flags { configAppend = v })
+      (boolOpt [] [])
+  , option "" ["overwrite"]
+      "the backup of the config file before any alterations"
```

Comment:
This is not what the flag does. In fact, it's the opposite.

Rather than fixing the help text, I suggest changing `overwrite` to `backup` and negating its current effect, since _backups_ are what the flag is actually controlling, and `overwrite` looks like the opposite of `append`.

Let's discuss it in the rfc: https://github.com/haskell/cabal/issues/7405#issuecomment-851697157

Reply:
I agree with your stance and made the alterations.